cornell-dti/idol
# Update product pages

### Summary 

This PR updates the /projects page and each individual product page on the DTI website. Furthermore, inactive projects have been removed from the website. 

The currently active projects should be:
- Courseplan
- CU Reviews
- QMI
- Carriage
- Zing
- D@C
- Clubview
- CU Apts 

### Test Plan 
- Check `/projects` to ensure that each of project listed above is on the page
-  Check each product's page for any typos, missing images, etc. (anything that looks out of place).
- Make sure there are no missing images for any of the products 
- **Make sure all images load reasonably quickly**

### Notes 
[FIXED] Some of the images need to be compressed -- there are a handful of images in the MBs that is likely causing them to load very slow
